assets/data/effects/drauven/drauven_pc_quickblade.json

```diff
@@ -36,7 +37,7 @@
 			"outcomes": [
 				{
 					"class": "Damage",
-					"amount": "(4+1d2)+(((self.PHYSICAL_DAMAGE_BONUS+self.POTENCY)*0.5)+(self.drauvenDeck_hunter_quickblade_upgrade*3))",
+					"amount": "(4+1d2)+(((self.PHYSICAL_DAMAGE_BONUS+self.POTENCY)*0.5)+(self.drauvenDeck_hunter_quickblade_upgrade*2))",
```

Comment:
🤔 Just as a point of comparison, the wolf bite is 2+1d3 + (half damage+potency) as the base damage. We might want to tone this damage down to be more in line with that skill, especially since shred and pierce are being stacked on top as well.